src/austin.c

```diff
@@ -195,6 +235,11 @@ int main(int argc, char ** argv) {
     }
   }
 
+  if (output_file != stdout) {
+    fclose(output_file);
```

Comment:
This leave the process without any stdout file descriptor which is really unusual. Every Unix program expects to have something in stdout/stderr/stdin:

```sh
$ ./src/austin echo test
echo: write error: Bad file descriptor
```

It'd be better to open `/dev/null` and dup to it stdout.

Reply:
Fixed